Day-5-Exercises/main.js

```diff
@@ -0,0 +1,217 @@
+
+
+// Slice out the middle IT company or companies from the array
+let center = Math.floor(itCompanies.length / 2)
+itCompanies.length % 2
+?console.log(itCompanies.slice(center, center + 1))
+:console.log(itCompanies.slice(center - 1, center + 1))
+
+// Remove the first IT company from the array
+itCompanies.shift()
+console.log(itCompanies)
+
+// Remove the middle IT company or companies from the array\
+center = Math.floor(itCompanies.length / 2)
+itCompanies.length % 2
+?itCompanies.splice(center, 1)
+:itCompanies.splice(center - 1, 2)
+console.log(itCompanies)
+
+// Remove the last IT company from the array
+itCompanies.pop()
+console.log(itCompanies)
+
+// Remove all IT companies
+itCompanies.splice(0,itCompanies.length)
+console.log(itCompanies,'The End')
+
+
+//Exercise: Level 2
+
+//Create a separate countries.js file and store the countries array in to this file, create a separate file web_techs.js and store the webTechs array in to this file. Access both file in main.js file
+
+//First remove all the punctuations and change the string to array and count the number of words in the array
+let text = 'I love teaching and empowering people. I teach HTML, CSS, JS, React, Python.'
+const words = text.replace(/[.,]/g,'').split(' ')
+console.log(words)
+console.log(words.length)
+
+//In the following shopping cart add, remove, edit items
+/*
+add 'Meat' in the beginning of your shopping cart if it has not been already added
+add Sugar at the end of you shopping cart if it has not been already added
+remove 'Honey' if you are allergic to honey
+modify Tea to 'Green Tea'
+*/
+const shoppingCart = ['Milk', 'Coffee', 'Tea', 'Honey']
+if (!shoppingCart.includes('Meat')) {shoppingCart.unshift('Meat')}
+if (!shoppingCart.includes('Sugar')) {shoppingCart.push('Sugar')}
+shoppingCart.splice(shoppingCart.indexOf('Honey'),1)
+shoppingCart[shoppingCart.indexOf('Tea')] = 'Green Tea'
+console.log(shoppingCart)
+
+//In countries array check if 'Ethiopia' exists in the array if it exists print 'ETHIOPIA'. If it does not exist add to the countries list.
+let newCountry = 'Ethiopia'
+countries.includes(newCountry)?console.log(newCountry.toUpperCase()):countries.push(newCountry)
+console.log(countries)
+
+//In the webTechs array check if Sass exists in the array and if it exists print 'Sass is a CSS preprocess'. If it does not exist add Sass to the array and print the array.
+let newTech = 'Sass'
+webTechs.includes(newTech)?console.log(`${newTech} is a CSS preprocess`):webTechs.push(newTech)
+console.log(webTechs)
+
+//Concatenate the following two variables and store it in a fullStack variable.
+const frontEnd = ['HTML', 'CSS', 'JS', 'React', 'Redux']
+const backEnd = ['Node','Express', 'MongoDB']
+
+const fullStack = frontEnd.concat(backEnd)
+
+console.log(fullStack)
+
+
+//Exercise: Level 3
+
+//The following is an array of 10 students ages:
+const ages = [19, 22, 19, 24, 20, 25, 26, 24, 25, 24]
```

Comment:
try adding 119 to the end of the array, and then sorting. It'll be interesting :)

Reply:
Reworked to sort as integers